guides/shipments/shipping.md

```diff
@@ -1,129 +1,194 @@
+- `cost`: The order cost for the select shipping method.
+- `order_id`: The ID for the order that a shipment is being created for.
 
-## Definitions, Design, & Functionality
+<!-- TODO:
+  Add a shipment process flow diagram.
+-->
 
-To properly leverage Solidus' shipping system's flexibility you must understand a few key concepts:
+### Shipping states
 
-* Stock Locations
-* Shipping Methods
-* Zones
-* Shipping Categories
-* Calculators (through Shipping Rates)
+Each shipment is assigned a `state` attribute. Depending on its state, different
+actions can be performed on shipments. There are four possible states:
 
-### Stock Locations
-Stock locations represent physical storage locations from which stock is shipped.
+- `pending`: The shipment has backordered inventory units and/or the order is
+  not paid for.
+- `ready`: The shipment has no backordered inventory units and the order is paid
+  for.
+- `shipped`: The shipment is on its way to the buyer.
+- `canceled`: When an order is cancelled, all of its shipments will also be
+  cancelled. When this happens, all items in the shipment will be restocked. If
+  an order is "resumed", then the shipment will also be resumed.
 
-### Shipping Methods
+## Core concepts
 
-Shipping methods identify the actual services or carriers used to ship the product. For example:
+To leverage Solidus's shipping system, become familiar with its key concepts:
 
-* UPS Ground
-* UPS One Day
-* FedEx 2Day
-* FedEx Overnight
-* DHL International
+- Stock locations
+- Shipping methods
+- Zones
+- Shipping categories
+- Shipping calculators (through shipping rates)
 
-Each shipping method is only applicable to a specific geographic **Zone**. For example, you wouldn't be able to get a package delivered internationally using a domestic-only shipping method. You can't ship from Dallas, USA to Rio de Janeiro, Brazil using UPS Ground (a US-only carrier).
+### Stock locations
 
-If you are using shipping categories, these can be used to qualify or disqualify a given shipping method.
+Stock locations represent physical storage locations from which stock is
+shipped.
 
-**Note**: *Shipping methods can have multiple shipping categories assigned to them. This allows the shipping methods available to an order to be determined by the shipping categories of the items in a shipment.*
+### Shipping methods
+
+Shipping methods identify the actual delivery services used to ship the
+product. For example:
+
+- UPS Ground
+- UPS One Day
+- FedEx 2Day
+- FedEx Overnight
+- DHL International
+
+Each shipping method is only applicable to a specific geographic
+zone. For example, you wouldn't be able to get a package delivered
+internationally using a domestic-only shipping method. You can't ship from
+Dallas, USA, to Rio de Janeiro, Brazil, using UPS Ground, which is a United
+States-only carrier.
+
+### Shipping categories
+
+You can further restrict when shipping methods are available to customers by
```

Comment:
Shipping categories can also be used to force items to ship separately. This is arguably their most important use case.